TrackletAlgorithm/VMRouterCMTop.h

```diff
@@ -0,0 +1,83 @@
+#ifndef TrackletAlgorithm_VMRouterCMTop_h
+#define TrackletAlgorithm_VMRouterCMTop_h
+
+#include "VMRouterCM.h"
+
+// VMRouter Top Function for layer 2, AllStub region A
+// Sort stubs into smaller regions in phi, i.e. Virtual Modules (VMs).
+
+// NOTE: to run a different phi region, change the following
+//          - constants specified in VMRouterCMTop.h
+//          - add/remove pragmas depending on inputStubs in VMRouterCMTop.cc
+//          - maskASI in VMRouterCMTop.cc
+//          - add the phi region in emData/download.sh, make sure to also run clean
+
+
+////////////////////////////////////////////
+// Variables for that are specified with regards to the VMR region
+// Changed manually
+
+#define kLAYER 2 // Which barrel layer number the data is coming from
+#define kDISK 0 // Which disk number the data is coming from, 0 if not disk
+
+constexpr char phiRegion = 'A'; // Which AllStub/PhiRegion
+constexpr int sector = 4; //  Specifies the sector
```

Comment:
I think this should be moved to the test bench, since it doesn't have anything to do with defining the top function.